otamodel/OTA_HotelProductNotifRQ.go

```diff
@@ -61,35 +81,34 @@ type HotelProductRatePlan struct {
 	Descriptions  Descriptions    `json:"Descriptions,omitempty,required"`
 	StayDateRange []StayDateRange `json:"StayDateRange,omitempty,required"`
 }
+
+//HotelProductRatePlans **
 type HotelProductRatePlans struct {
 	RatePlan *[]HotelProductRatePlan `json:"RatePlan,omitempty,required"`
 }
+
+//PrepaymentPolicy **
 type PrepaymentPolicy struct {
 	EffectiveFrom string `json:"EffectiveFrom,omitempty"`
 }
+
+//TPAExtensions **
 type TPAExtensions struct {
 	PrepaymentPolicy PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
 }
-type GuaranteePayment struct {
-	AcceptedPayments *AcceptedPayments `json:"AcceptedPayments,omitempty"`
-	TPAExtensions    *TPAExtensions    `json:"TPA_Extensions,omitempty"`
-	PolicyCode       string            `json:"PolicyCode,omitempty"`
-}
-
-type AcceptedPayments struct {
-	AcceptedPayment []AcceptedPayment `json:"AcceptedPayment"`
-}
 
-type AcceptedPayment struct {
-	PaymentCard PaymentCard `json:"PaymentCard"`
+//GuaranteePayment **
+type GuaranteePayment struct {
```

Comment:
why you have modified this struct.
please check OTA doc. this is not right struct as per OTA pilot fish doc

Reply:
Done